Userland/Libraries/LibJS/Tests/parser-unary-associativity.js

```diff
@@ -12,3 +12,7 @@ test("basic functionality", () => {
     expect((typeof "x" === "string") === true).toBeTrue();
     expect(!(typeof "x" === "string") === false).toBeTrue();
 });
+
+test("unary +/- operators bind higher than binary", () => {
+    expect(10 ** -3 / 2).toEqual(0.0005);
```

Comment:
Without this patch, this expression evaluates to `0.03162277660168379` !

Reply:
Whoops! Nice catch :)